src/DynamoCore/Utilities/UpdateValueParams.cs

```diff
@@ -0,0 +1,18 @@
+﻿using ProtoCore.Namespace;
+
+namespace Dynamo.Utilities
+{
+    public class UpdateValueParams
+    {
+        public UpdateValueParams(string value, string propertyName, ElementResolver elementResolver = null)
```

Comment:
Uh oh, I think there's a slight problem. The exact order of the argument must be 'propertyName', followed by 'propertyValue' (though it does not make a difference but from the API design standpoint I'd need it to be that way):

```csharp
/// <summary>
/// Creates an instance of UpdateValueParams object.
/// </summary>
/// <param name="propertyName">Name of the property whose value is to be updated.
/// This parameter cannot be empty or null.</param>
/// <param name="propertyValue">Value of the named property whose value is to be 
/// updated. This parameter can either be null or empty if the targeted property 
/// allows such values.</param>
/// 
public UpdateValueParams(string propertyName, string propertyValue,
    ElementResolver elementResolver = null)
{
}
```